app/controllers/admin/topics_controller.rb

```diff
@@ -312,6 +312,34 @@ def assign_team
     end
   end
 
+  def split_topic
+    parent_topic = Topic.find(params[:topic_id])
+    parent_post = Post.find(params[:post_id])
+
+    topic = parent_topic.forum.topics.new(
+      name: "Split from #{parent_topic.name}",
+      user: parent_topic.user
+    )
```

Comment:
Lets set the `forum_id` to 1 here (which makes the topic private.)  This feature will be used exclusively for splitting private tickets so I don't think it makes sense.